### DISCUSSION TITLE
Why are you forking FireCMS?

### DISCUSSION THREAD

**Discussion by fgatti675** [firecmsco/firecms#330] [Polls]
Hi everyone, it is very interesting to see how often FireCMS is being forked.

The library is meant to provide multiple expansion points that are intended not to make forking necessary.
At the time of writing the ratio is 5 stars to 1 fork approximately, which is higher than in any other open source project I have seen.

Forking is a core concept of open source and it is great, but it makes that some developments that could be useful for everyone may get lost!
So I'd like to start a little discussion and try to understand better the case :)

**Comment by JohnFromme** [comment]
Yes, you are right! Forking is not necessary. Everything I needed to customize for my project already came built into FireCMS which is super awesome! It is very easy to customize FireCMS without needing to directly edit the package code.
